examples/axios/pages/[user]/[repo].js

```diff
@@ -0,0 +1,39 @@
+import Link from 'next/link'
+
+import useRequest from '../../libs/useRequest'
+
+export default () => {
+  const id =
+    typeof window !== 'undefined' ? window.location.pathname.slice(1) : ''
+  const { data } = useRequest(
+    id
+      ? {
+          url: '/api/data',
+          params: {
+            id
+          }
+        }
+      : null,
+    fetch
```

Comment:
Is this `fetch` needed?

Reply:
Nope, and isn't even defined, so not sure why it didn't complain here 😕 Removed!